cart-service/.dockerignore

```diff
@@ -0,0 +1,10 @@
+# ignore .git and .cache folders
+.git
+.cache
+# ignore .test dolder, linter & prettier configs
+.test
+.eslint*
+.prettier*
+.README.md
+# ignore .node_modules
+.node_modules
```

Comment:
you also can add `dist` folder here, as far as it will be generated inside of container

Reply:
thank you, fixed